assets/js/components/Config/DataManagementModal.vue

```diff
@@ -0,0 +1,149 @@
+<template>
+	<GenericModal
+		id="dataManagementModal"
+		:title="$t('config.system.dataManagement.title')"
+		data-testid="data-management-modal"
+		@opened="$emit('opened')"
+		@closed="$emit('closed')"
+	>
+		<p>
+			<span>{{ $t("config.system.dataManagement.description") }}</span>
+		</p>
+
+		<div>
+			<h6>
+				{{ $t("config.system.dataManagement.backup.title") }} <small>backup-summary</small>
+			</h6>
+			<button class="btn btn-outline-secondary" @click="openBackupConfirmModal">
+				{{ $t("config.system.dataManagement.backup.download") }}
+			</button>
+		</div>
+		<div>
+			<h6>
+				{{ $t("config.system.dataManagement.restore.title") }} <small>backup-restore</small>
+			</h6>
+
+			<PropertyFileField
+				ref="fileInput"
+				:accepted="['.db']"
+				required
+				@file-changed="fileChanged"
+			/>
+
+			<button
+				class="btn btn-outline-secondary mt-2"
+				:disabled="file === null"
+				@click="restoreDatabase"
+			>
+				{{ $t("config.system.dataManagement.restore.action") }}
+			</button>
```

Comment:
Eingabefeld und Buttons sollten zusammen in ein Form. Darüber dann auch die Aktion triggern. Siehe unten.